docs/admin/ha-master-gce.md

```diff
@@ -0,0 +1,159 @@
+---
+assignees:
+- jszczepkowski
+
+---
+
+* TOC
+{:toc}
+
+## Introduction
+
+In kubernetes version 1.5, we added alpha support for replication of kubernetes masters in kube-up/down scripts for GCE.
+This document describes how to use kube-up/down scripts to manage highly available (HA) masters and how HA masters are implemented for GCE case.
+
+## Running HA cluster on GCE
+
+### Starting HA-compatible cluster
+
+When creating a new HA cluster, two flags need to be set for kube-up script:
```

Comment:
"To create a new HA cluster, you must set the following flags in your `kube-up` script:"